.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         if: steps.check-for-backend.outputs.has-backend == 'true'
         uses: actions/setup-go@v3
```

Comment:
Could you bump all instances of `setup-go` to `v5`?

Reply:
```suggestion
        uses: actions/setup-go@v5
```